csharp_dotnetcore/51.Enterprise-Bot-Template/CogSvcModels/Dispatch/Dispatch.cs

```diff
@@ -0,0 +1,64 @@
+// <auto-generated>
```

Comment:
CogSvcModels should be CongnitiveModels. Also unclear why you have .cs files under cognitiveModels.

Reply:
I wasn't quite sure where the LG files should go once they are created so I put them in CognitiveModels. Thoughts on where they should go?